modules/how-to-plan-your-environment-according-to-application-requirements.adoc

```diff
@@ -167,3 +167,22 @@ links in the deployment's service specification file to overcome this:
     labels:
       template: deploymentConfigTemplate
 ----
+
+The number of application pods that can run in a namespace is dependent on the number of services and the
+length of the service name when the environment variables are used for service discovery. ARG_MAX on the system 
+defines the maximum argument length for a new process and it it set to 2097152 KiB by default. Kubelet injects 
+environment variables in to each pod scheduled to run in the namespace including:
+
+----
```

Comment:
Because this code block is a list of variables and not example output, I would take it out of the codeblock and create an unordered list with the variables in back-ticks.